hazelcast-code-generator/src/main/resources/codec-template-java.ftl

```diff
@@ -26,7 +26,7 @@ public final class ${model.className} {
         public static int calculateDataSize(<#list model.requestParams as param><@methodParam type=param.type/> ${param.name}<#if param_has_next>, </#if></#list>) {
             int dataSize = ClientMessage.HEADER_SIZE;
 <#list model.requestParams as p>
-    <@sizeText varName=p.name type=p.type isNullable=p.nullable/>
+    <@sizeText varName=p.name type=p.type isNullable=p.nullable maybeNullElements=p.mayContainNullElements/>
```

Comment:
lets use the names consistent, using both maybeNullElements and mayContainNullElements is confusing.